rainforest/reporter.go

```diff
@@ -0,0 +1,103 @@
+package rainforest
+
+import "strconv"
+
+// RunFeedback contains details about the feedback of a Run Step for a browser
+type RunFeedback struct {
+	AnswerGiven string `json:"answer_given"`
+	JobState    string `json:"job_state"`
+	Note        string `json:"note"`
+}
+
+// RunBrowserDetails contains details about a Browser of a Run Step
+type RunBrowserDetails struct {
+	Name     string        `json:"name"`
+	Feedback []RunFeedback `json:"feedback"`
+}
+
+// RunStepDetails contains details about a Run Step
+type RunStepDetails struct {
+	Browsers []RunBrowserDetails `json:"browsers"`
+}
+
+// RunTestDetails contains details about a Run Test
+type RunTestDetails struct {
+	ID        int              `json:"id"`
+	Title     string           `json:"title"`
+	CreatedAt string           `json:"created_at"`
```

Comment:
I thought of this. I forgot why I decided against... Doesn't seem to be a good reason I can think of.